website/docs/enterprise/upgrading.mdx

```diff
@@ -84,15 +158,35 @@ The example templates provided in this guide have been tested with `clusterctl`
 Download a specific version of clusterctl from the [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
 :::
 
-In order to be able to provision Kubernetes clusters, a CAPI provider needs to be installed.
+In order to be able to provision Kubernetes clusters, a CAPI provider needs to be installed. See [Cluster API Providers](../cluster-management/cluster-api-providers.mdx) page for more details on providers.
+He're we'll continue with our example instructions for CAPD and CAPA.
```

Comment:
Should this be `Here` ?

Reply:
Good catch, the internet concurs that _He're_ is not a valid contraction of anything.

Reply:
```suggestion
Here we'll continue with our example instructions for CAPD and CAPA.
```